language-server/src/server.js

```diff
@@ -259,9 +300,16 @@ const getTokenBuilder = (uri) => {
   return result;
 };
 
-const buildTokens = (builder, document) => {
+const buildTokens = (builder, document, settings) => {
   const instance = JsoncInstance.fromTextDocument(document);
-  const dialectUri = instance.get("#/$schema").value();
+  const $schema = instance.get("#/$schema");
+  let dialectUri;
+  try {
+    dialectUri = $schema?.value() ?? settings.defaultDialect;
+  } catch (error) {
+    // SKIP
+  }
```

Comment:
I think the error you were encountering here was caused by a bug that was fixed in JsoncInstance and when you rebased, you got the fix and this hack to catch the error was no longer necessary. The following works for me.

```suggestion
  const dialectUri = $schema.value() ?? settings.defaultDialect;
```

Reply:
Oh okay, This was the initial approach. But had to use the hack before rebasing, works now.

Reply:
I did commit the code but later realise that this is the semantic token highlighting when I do not use the hack:
![image](https://github.com/hyperjump-io/json-schema-language-tools/assets/110971977/87375dff-c10b-4d34-868a-b36f26f80694)


and below is the semantic highlighting when I use the hack.
![Uploading image.png…]()

Reply:
@jdesrosiers So, Should I revert to using the hack or try finding an alternative to semantic token highlighting.

Reply:
You have a typo. The `value` method is not being called. (You missed the `()`)